torch_xla/csrc/ops/dynamic_ir.cpp

```diff
@@ -95,7 +98,10 @@ XlaOpVector SizeAdd::Lower(LoweringContext* loctx) const {
 
 SizeEq::SizeEq(torch::lazy::Value a, torch::lazy::Value b)
     : XlaNode(torch::lazy::OpKind{c10::Symbol::fromQualString("aten::eq")},
-              {a, b}, xla::ShapeUtil::MakeShape(xla::S64, {}), 1) {
+              {a, b},
+              xla::ShapeUtil::MakeShape(
```

Comment:
In my original problematic [pr](https://github.com/pytorch/xla/pull/4410), it uses `GetShapeDimensionType(/*device=*/nullptr)` which only uses a `xla::PrimitiveType` but the base class constructor needs a `shape`. So the difference here is I use `xla::ShapeUtil::MakeShape` to pass a `shape` to base class constructor.